samigo/samigo-app/src/webapp/jsf/author/authorSettings.jsp

```diff
@@ -84,13 +84,15 @@
               }
 
               if (enabled) {
+            	  $('#assessmentSettingsAction\\:linear_access_warning').hide();
                   $('#assessmentSettingsAction\\:markForReview1').removeAttr("disabled");
                   $('#assessmentSettingsAction\\:markForReview1').parent().removeClass("placeholder");
                   QuesFormatRadios.forEach( function(v, i, a) {
                       $('label[for="' + v + '"]').removeClass("placeholder");
                       $("#" + v).removeAttr("disabled");
                   });
               } else {
+            	  $('#assessmentSettingsAction\\:linear_access_warning').show();
                   $('#assessmentSettingsAction\\:markForReview1').attr("disabled", true);
                   $('#assessmentSettingsAction\\:markForReview1').prop("checked", false);
                   $('#assessmentSettingsAction\\:markForReview1').parent().addClass("placeholder");
```

Comment:
I know this is how it was before, but you could just chain all these calls onto the one jquery object.

Reply:
Hi @adrianfish I'm not sure I understand what you mean. 
Which calls do you refer? Do you refer to chain calls to "markForReview1"? 
I have introduced just the "linear_access_warning" lines, and there is one call onlly

Reply:
Yes, just to clean up the code a bit and make it a bit performant. I know it wasn't your code, but it's always nice to do a bit of cleanup around the area you're fixing.

$('#assessmentSettingsAction\\:markForReview1')
  .attr("disabled", true)
  .prop("checked", false)
  .parent().addClass("placeholder");